GliaWidgets/Coordinator/RootCoordinator.swift

```diff
@@ -1,5 +1,5 @@
-import UIKit
 import SalemoveSDK
+import UIKit
```

Comment:
How do you order them? Alphabetically?

Reply:
Ah, this was actually SwiftFormat who moved them around and I didn't notice. [Alphabetically indeed.](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#sortedImports)